accepted/0000-timezones.md

```diff
@@ -0,0 +1,103 @@
+- Feature Name: timezones
+- Start Date: 2019-09-16
+- RFC PR: 
+
+# Summary
+[summary]: #summary
+
+Timezones can be a big problem when managing an application like Uyuni that can be deployed and accessed from anywhere in the world. 
+If we don’t have some rules on how to display and handle dates, this can become quite a big mess to manage. 
+
+
+# Motivation
+[motivation]: #motivation
+
+The goal of this RFC is to define those rules and serve as a reference for the future.
+
+These rules should define a consistent way of display dates on the UI and make this information as clear as possible for the user.
+
+The user should be able to easily understand in which timezone a date is displayed and aware if there is any difference from the timezone
+he is accessing from.
+
+# Glossary
+[glossary]: #glossary
+
+When talking about uyuni there are 4 different Timezones contexts that are important to understand and distinguish:
+
+* **Managed systems timezone:** the timezone of each minion/client. This doesn't have any impact on this RFC, but it's important to mention.
+
+* **Uyuni Server timezone:** the configured timezone in the server where uyuni is deployed 
+
+* **User preferences timezone:** the defined timezone on the user preferences settings. Here we should be able to choose the timezone, date format (yyyyddmm) and the time format (12/24 hours).
+All the shown dates on the UI and operations on them should always assume these user settings. 
```

Comment:
Does this include the API? I can see we have several API calls where datetime is involved in ISO 8601 format but we only talk about YYYYMMDD HHMM, no timezone. E. g. scheduleChain, scheduleImageBuild, listAllPackages, etc.

Reply:
To be honest my knowledge about the XMLRPC api is a bit limited. I was focusing the RFC more on the UI part. 

Do we have any problem with the API regarding timezones?

For me, it would make sense if we return the timezone information on the API as well, so no context is lost. Although I am not sure if this can be a breaking change.

Is Dario answer enough here?

> XMLRPC API endpoints should use and return datetime based on the user settings the one who is executing the API function. For the Salt command line AFAIK it can be used only on the server as root so, it datetime should be the one that the server has.